index.html

```diff
@@ -19,60 +14,89 @@
+            </div>
+            <div class="row"></div>
+            <div id="wordList">
+            <div class="row noMargin hide wordAndScoreRow">
+                <div class="col s9 noPadding">
+                    <h6 class="word"></h6>
+                </div>
+                <div class="col s3 noPadding right-align">
+                    <label class="white-text orange labelPadding wordScore"></label>
+                </div>
+            </div>
+            </div>
+                <div id="codeLink"><a target = "_blank" href="https://github.com/eastnorthwest/convoluted-butterfly"> See the code</a></div>
 		</div>
 		<div class="col s9 fullHeight" id="right">
 			<div id="rightColumns">
-				<div class="row col s12"> </div>
+                <div class="row col s12"></div>
 				<div class="row">
 					<div class="col s2"></div>
-					<div class="col s8 center-align">Code</div>
-					<div class="col s2"><button class="btn light-green">Submit</button><br/><button class="btn red">Clear</button></div>
+					<div class="col s8 center-align"><h2 id="playWord"></h2></div>
+                    <div class="col s2"><button class="btn light-green controls" id="submit">Submit</button><br/><button class="btn red controls" id="clear">Clear</button></div>
+                </div>
 				<div class="row">
 					<div class="col s2"></div>
-					<div class="col s8  center-align red">	
-						<div>			
-						<div class="row centeringSquares"> 
-							<div class="col square yellow valign-wrapper center-align" >S</div>
-							<div class="col square yellow valign-wrapper center-align">D</div>
-							<div class="col square yellow valign-wrapper center-align">L</div>
-							<div class="col square yellow valign-wrapper center-align">G</div>
-						</div>
-						<div class="row centeringSquares"> 
-							<div class="col square yellow valign-wrapper center-align" >S</div>
-							<div class="col square yellow valign-wrapper center-align">D</div>
-							<div class="col square yellow valign-wrapper center-align">L</div>
-							<div class="col square yellow valign-wrapper center-align">G</div>
-						</div>
-						<div class="row centeringSquares"> 
-							<div class="col square yellow valign-wrapper center-align" >S</div>
-							<div class="col square yellow valign-wrapper center-align">D</div>
-							<div class="col square yellow valign-wrapper center-align">L</div>
-							<div class="col square yellow valign-wrapper center-align">G</div>
-						</div>	
-						<div class="row centeringSquares"> 
-							<div class="col square yellow valign-wrapper center-align" >S</div>
-							<div class="col square yellow valign-wrapper center-align">D</div>
-							<div class="col square yellow valign-wrapper center-align">L</div>
-							<div class="col square yellow valign-wrapper center-align">G</div>
-						</div>	
-						</div>											
+					<div class="col s8  center-align">
+						<div>
+                            <div class="row centeringSquares">
```

Comment:
It would appear by using the github code review tool that there are weird whitespace blocks. I may be wrong, or you may be using them for some reason, but if you aren't please remove.